daemon/algod/api/server/v2/generated/participating/private/private_routes.yml

```diff
@@ -0,0 +1,16 @@
+package: private
+generate:
+  echo-server: true
+  embedded-spec: true
+output-options:
+  include-tags:
+  - participating
```

Comment:
@Eric-Warehime Request for clarification:  Should I expect to see tag = `private` _included_ in these yml definitions?

I only see it excluded.  And I feel like I'm not following the intuition, so I'm asking the question.

Reply:
I guess this is because endpoints now have multiple tags, so this is implicitly selecting endpoints which have `participating && !public`, which would include endpoints with `participating && (private || common)`.

Reply:
@Eric-Warehime What about scenarios where the endpoint _only_ specifies tag = `private`?

* Path for `AbortCatchup` is one such example.  It's possible the path is a misconfiguration.  Though that might be a reason to fully qualify includes and excludes.
* Here's an example patch that generates a different outcome than the PR:  [diff.txt](https://github.com/algorand/go-algorand/files/9974980/diff.txt).

Reply:
After our conversation, I've added tags to AbortCatchup to make sure it's included in the Router definitions. And I've updated all of the config files to exhaustively list the set of excluded tags as you've done in your example diff.

Reply:
@Eric-Warehime Resolving - Thanks for the changes in https://github.com/algorand/go-algorand/pull/4714/commits/0171f8e6b713aa4e0ad206d743b00884e9bf12c2 and https://github.com/algorand/go-algorand/pull/4714/commits/f4bd5fa83dc72d5f1a9438fc6507febb6bb98b1b.

Leaving a note for future thought:  
* Prior to https://github.com/algorand/go-algorand/pull/4714/commits/f4bd5fa83dc72d5f1a9438fc6507febb6bb98b1b, is there no compile error.
* If we'd like to avoid such regressions in the future, one approach is to modify tests to use the generated interface rather than using the concrete type (`v2.Handlers`).  If tests like `abortCatchupTest` relied on the interface, I think we'd catch the regression at compile time.
* I didn't investigate.  My surface level understanding is: